fastestimator/util/traceability_util.py

```diff
@@ -1080,3 +1085,33 @@ def fe_state(self) -> Mapping[str, Any]:
         return cls
 
     return make_traceable
+
+
+def get_environment() -> dict:
```

Comment:
revert this since no longer being used in the new trace

Reply:
fixed, now traceability.py is reverted